src/view3d.cpp

```diff
@@ -448,92 +650,205 @@ void View3D::drawCursor(const TransformState& transStateIn, const V3d& cursorPos
     TransformState transState = transStateIn.translate(offset);
     V3d relCursor = cursorPos - offset;
 
-    // Cull if behind camera
-    if((relCursor * transState.modelViewMatrix).z > 0)
-        return;
+    // Cull if behind camera -> doesn't work for now
+    //if((relCursor * transState.projMatrix).z > 0)
+    //    return;
```

Comment:
Any commentary here on what exactly was going wrong?  Did it work before and not now?  Was it just plainly broken before (entirely possible)?